src/en/space-station-14/departments/security/proposals/reduced-metagaming.md

```diff
@@ -0,0 +1,234 @@
+
+- Items that can't be found unless directly used
+  - Conducting Gloves
+  - Thieving Gloves
+  - Hypodart
+  - Decoy Nuclear Disk
+  - Explosive Banana
+  - Explosive Wet Floor Sign
+ 
+- Items that can be found if worn
+  - Voice Mask
+  - Chameleon Kit
+  - No-slip Shoes
+
+- Items that can be found if held
+  - Hypopen
+  - Stimkit
+  - Agent ID Card
+  - Lobbying Bundle
+  - Energy Dagger
+  - Cane Sword
+
+- Items that can be found if examined
+  - Extra-Bright Lantern
+  - Blue Boxing Gloves
+  - Dehydrated Space Carp
+
+There is a question to be made what the intended purpose of these disguises are. Something like the Extra-Bright Lantern can only survive a passing glance from crew, a cane sword may be found if an officer is searching thoroughly and a hypodart can't be determined to be a Syndicate object unless explicitly tested. The problem arises when Security is able to be too thorough with a search and find an item they aren't looking for, or when Security knows for certain an item is contraband but can't prove it.
+
+There is also an issue where these disguised items are unknown to crew (until an uplink is found), while the other "obvious" antag items are not. So [crew can have full knowledge](https://forum.spacestation14.com/t/what-is-a-syndicate-item/5363) of emags, webvests, e-swords and northstars, but not about hypopens or thieving gloves. 
+
+### Proposed Solution
+
+This is probably the most comprehensive change and touches on things outside the stated problem above, in the interest of making a more streamlined system and help onboarding new players into contraband gameplay. My proposal is the following:
+
+- Rule change: Crew have knowledge about stealth items.
+- *Non-stealth* items can now be determined to be contraband upon being examined.
+  - Improvised contraband (improvised weapons, shivs, makeshift cuffs etc.) have the following text added: "This item is minor contraband."
+  - Items restricted to departments (insulated gloves, RCDs, scalpels, Security gear etc.) have the following text added: "This item is departmentally restricted."
+  - Command items (HoS' secret orders, Captain's Sabre, Nuke Disk, CE's boots etc.) have the following text added: "This item is restricted to Command."
+  - Syndicate items have the following text added: "This item is Syndicate contraband."
+- Note that the change above does not change the crime definitions in Space Law.
+- Most stealth items can now be locked to hide their functionality, making it act as a normal item. If the wearer/holder speaks a Traitor codeword, the functionality unlocks.
+  - While locked, the contraband examine text is not shown. 
+  - Thieves get a single codeword, not shared by other thieves or traitors.
+  - Nukies get a set codeword "Syndicate".
+  - Note that this is not applicable for all items. For example, the Extra-Bright Lantern can not be locked, allowing it to only pass inspection at a glance.
```

Comment:
This is a good idea :)

(i figure I should at least note approval of some of these things after railing so hard on it)